src/scripts/webact.js

```diff
@@ -224,53 +206,53 @@ function getTabsFromStorage(tabs) {
 
     ui.clearUI();
     if (tabs === null) {
-      ui.fillEmptyBlock("chart");
-      return;
+        ui.fillEmptyBlock("chart");
+        return;
     }
 
     var counterOfSite;
     if (currentTypeOfList === TypeListEnum.All) {
         targetTabs = tabs;
-      if (targetTabs.length > 0) {
-        totalTime = getTotalTime(targetTabs, currentTypeOfList);
-        stat.allDaysTime = totalTime;
-      } else {
-        ui.fillEmptyBlock("chart");
-        return;
-      }
+        if (targetTabs.length > 0) {
+            totalTime = getTotalTime(targetTabs, currentTypeOfList);
+            stat.allDaysTime = totalTime;
+        } else {
+            ui.fillEmptyBlock("chart");
+            return;
+        }
 
-      counterOfSite = tabs.length;
+        counterOfSite = tabs.length;
```

Comment:
If this variable represents the number of sites, the better name for it is `siteCount`